tests/unit/lms/db/__init___test.py

```diff
@@ -1,41 +0,0 @@
-import sqlalchemy as sa
-
-from lms.db import BASE
-
-
-class Child(BASE):
-    __tablename__ = "child"
-    id = sa.Column(sa.Integer, primary_key=True)
-
-
-class ModelClass(BASE):
-    __tablename__ = "model_class"
-    _aliased_column = sa.Column("aliased_column", sa.Integer)
-    id = sa.Column(sa.Integer, primary_key=True)
-    column = sa.Column(sa.Integer, sa.ForeignKey("child.id"))
-    relationship = sa.orm.relationship("Child")
-
-
-class TestBase:
-    def test_we_can_get_columns(self):
-        assert sorted(ModelClass.columns()) == [
-            "_aliased_column",
-            "column",
-            "id",
-        ]
-
-    def test_repr(self):
-        model = ModelClass(_aliased_column=77, id=23, column=46)
-
-        assert repr(model) == "ModelClass(_aliased_column=77, id=23, column=46)"
-
-    def test_repr_is_valid_python(self):
```

Comment:
Not sure what the idea behind making sure this is a property of the repr, :thinking:

Reply:
The test is a bit much. It's normal in Python for `__repr__()` strings to _look like_ executable Python code if possible, and if not then they fall back on a string wrapped in `<...>`'s instead:

> [If at all possible, this should **look like** a valid Python expression that could be used to recreate an object with the same value (given an appropriate environment). If this is not possible, a string of the form `<...some useful description...>` should be returned.](https://docs.python.org/3/reference/datamodel.html?highlight=__repr__#object.__repr__)

(Emphasis mine.)

If the `__repr__()` string really is executable Python code this can occasionally be handy when mucking around in a shell or debugger, to be able to copy some object that has been printed and paste it to create a new instance with the same values. But you'd rarely actually do that and it's often not possible or not desirable for the `__repr__()` to literally work like that. So I think the style is mostly just about readability.